Alerting: Introduces `/api/v1/ngalert/alertmanagers` to expose discovered and dropped Alertmanager(s) (#37632)
grafana/grafana: docs/sources/administration/configuration.md

Sets a global limit on number of alert rules that can be created. Default is -1 (unlimited).

<hr>

> ## [ngalerting]

Author: @torkelo @kylebrandt any thoughts?

> ## [ngalerting]

Author: we have settled on `unified-alerting`.

For more information about the NGAlert feature in Grafana, refer to [Unified Alerting]({{< relref "../unified-alerting/_index.md" >}}}).

### admin_config_poll_interval_seconds

> How frequently we poll for admin config changes. Default value is `60`.

Reviewer: ```suggestion
 Specify the frequency of polling for admin config changes. The default value is `60`.
```

Author: Thanks, will update.
